### ISSUE TITLE
Comet triggers before the triggering event finishes resolving

### ISSUE THREAD

**Issue by anathomical** [mtgred/netrunner#322]
Currently Comet asks if you want to play a second event as soon as you play the first event.  This causes problems if the first event played requires the player to further interact (via a run, or via other interaction prompts).

Comet sets up a parallel set of decisions to interact with, and those interactions alternate with the first set.  Here's an example:

1. Comet is installed.
2. Runner plays Test Run.  Is shown an interaction prompt asking if they want to play an additional event.
3. Runner selects "Yes".  Is shown an interaction prompt asking if they want to use Test Run to search their Stack or their Heap.
4. Runner selects "Stack".  Is shown an interaction prompt asking what additional event to play with Comet.
5.  Runner selects "Infiltration".  Is shown an interaction prompt to select which program to install from the stack with Test Run.
6. Runner selects "Femme Fatale".  Is shown an interaction prompt to choose to gain 2 credits or expose a card.

Now, in this particular example, this isn't a huge problem, but if the runner wants to play Scavenge at step 5, the Femme Fatale isn't yet installed.  And this gets REALLY messed up if one (or worse, both) of the events triggers a run.  In fact, if they're both run events, the second run event gets triggered in the middle of the first run event, disrupting the run.

Basically, it would be cleaner if we could tell when an event is "finished resolving" so that Comet could trigger then rather than on-play.  That seems complicated, so it may make sense to de-automate Comet so that rather than prompting to play a second event when you play an event, the user can click on Comet to trigger the second event when appropriate.

**Comment by JoelCFC25** [comment]
This one is closely related to #183, but broader.

**Comment by gandrasch** [comment]
Problem still occours (Comet with Inject as example, I have to choose another event before gaining the 4 cards from inject).
